apps/api/src/controllers/user.controller.ts

```diff
@@ -515,9 +486,6 @@ export function makeUserController(
           id: userId,
           isSubscribed: true,
         });
-        if (!updatedUser) {
```

Comment:
updateUser always returns the user or throws, it never returns a null or undefined return value, so this check was completely unnecessary